src/pages/start-page/StartPage.tsx

```diff
@@ -49,8 +49,19 @@ const StartPage = () => {
     setShowError(true);
   };
 
+  const joinMeetingEnterKey = (event) => {
+    if (event.code === "Enter") {
```

Comment:
Also if you realize, you can reuse code here, just calling `joinMeeting()` inside the if statement that checks for ENTER code

<img width="259" alt="image" src="https://user-images.githubusercontent.com/26444448/179400934-72ef807f-6799-4155-b217-8bc6cc0e3aa5.png">